frontend/resourceadm/utils/resourceUtils/resourceUtils.ts

```diff
@@ -24,31 +32,30 @@ export const getResourcePageTextfieldError = (text: SupportedLanguageKey<string>
  *
  * @returns the string to display
  */
-export const convertResourceTypeToDisplayString = (resourceType: ResourceTypeOption) => {
-  if (resourceType === 'Default') return 'Standard';
-  else if (resourceType === 'Systemresource') return 'System ressurs';
-  else if (resourceType === 'MaskinportenSchema') return 'Maskinporten skjema';
-  return undefined;
+export const convertResourceTypeToDisplayString = (resourceType: ResourceTypeOption): string => {
+  return resourceTypeMap[resourceType];
 };
 
 /**
  * Maps the language key to the text
  */
-export const mapLanguageKeyToLanguageText = (val: 'nb' | 'nn' | 'en') => {
-  if (val === 'nb') return 'Bokmål';
-  if (val === 'nn') return 'Nynorsk';
-  return 'Engelsk';
+export const mapLanguageKeyToLanguageText = (val: 'nb' | 'nn' | 'en', translationFunction: any) => {
```

Comment:
What about typing the method instead of any? 😄 
```suggestion
export const mapLanguageKeyToLanguageText = (val: 'nb' | 'nn' | 'en', translationFunction: (key: string, params: object) => string) => {
```

Reply:
Good idea :) 
But i think the params are optional ?

Reply:
Yes, it should be optional. :)